magento/ece-tools
# MAGECLOUD-4970: Move all environment variables to schema config

### Description

https://magento2.atlassian.net/browse/MAGECLOUD-4970


### Manual testing scenarios

1. Run `php bin/ece-tools schema:generate` comnmand
2. Check that all configuration options exist in the dumped file `.magento.env.md`

### Contribution checklist
